src/cmsio0.c

```diff
@@ -485,6 +485,11 @@ cmsIOHANDLER* CMSEXPORT cmsGetProfileIOhandler(cmsHPROFILE hProfile)
 	return Icc->IOhandler;
 }
 
+#if !defined(HAVE_GMTIME_R) && !defined(HAVE__GMTIME64_S)
+// A global mutex used to serialize calls to gmtime.
+static _cmsMutex _cmsTimeMutex = CMS_MUTEX_INITIALIZER;
```

Comment:
would it be better to have this as a local static variable?

Reply:
Good question. According to https://stackoverflow.com/a/14107497, that would only be safe for C++11 and later versions (unless I misinterpreted that). I made it global static just to be sure.